Brewfile

```diff
@@ -0,0 +1,5 @@
+tap 'AdoptOpenJDK/openjdk'
+brew 'clang-format'
+brew 'node'
+brew 'yarn'
+cask 'adoptopenjdk11'
```

Comment:
I don't remember of a specific example when android toolchain breaks on JDK's other than v8, but it does happen fairly often. Hence, I would rather depend on v8 to avoid issues. 

Also JDK is often installed on the system, for example using sdk man. Is it possible to make this dependency optional? Installing multiple JDKs on the system can mess-up path variables.

Reply:
Don’t you need JRE installed to run sdkmanager in the first place? This is for machines that don’t have it installed to begin with. I have a MacBook at home where I recently did a clean install of Catalina. It didn’t have JRE installed, and this was the version that worked for me. Other versions crashed or threw exceptions.

Reply:
Tried `adoptopenjdk8` and it works, so I'll downgrade.